client/network/network.js

```diff
@@ -52,28 +52,16 @@ export function send(event, data) {
     socket.emit(event, data);
   }
 }
+
+// Return an object that can be opened to create an isolated per-module network,
+// and closed to clean up after that module.
 export function forModule(id) {
```

Comment:
Freaks me out that this per-module-network stuff is intermingled with the generic network stuff.
Not proposing any solutions, and finding none a-plenty.

Reply:
*this* part is not the scary one, because in this case, in the lcient binary, there's only ever 1 socket to worry about, so everything is easy: framework communication happens on the socket, module communication happens on this wrapped module overlay socket. The scary one is the equivalent in server/network/network.js.